package.json

```diff
@@ -41,53 +42,54 @@
     "diff-match-patch": "1.0.5",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-unicorn": "^39.0.0",
+    "eslint-plugin-unicorn": "^50.0.1",
     "font-awesome": "4.7.0",
     "jquery": "^3.6.0",
     "jqwidgets-ng": "^14.1.17",
-    "ngx-cookie-service": "^13.1.2",
-    "ngx-text-diff": "^0.6.0",
+    "monaco-editor": "0.45.0",
+    "ngx-cookie-service": "^15.0.0",
+    "ngx-monaco-editor-v2": "16.0.1",
     "path": "^0.12.7",
     "prettier": "2.5.1",
     "rxjs": "7.4.0",
     "tslib": "2.3.1",
     "xml-beautifier": "0.5.0",
-    "zone.js": "0.11.4"
+    "zone.js": "0.14.3"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^14.2.10",
-    "@angular-eslint/builder": "14.4.0",
-    "@angular-eslint/eslint-plugin": "14.4.0",
-    "@angular-eslint/eslint-plugin-template": "14.4.0",
-    "@angular-eslint/schematics": "14.4.0",
-    "@angular-eslint/template-parser": "14.4.0",
-    "@angular/cli": "14.2.10",
-    "@angular/compiler-cli": "14.2.12",
+    "@angular-devkit/build-angular": "^17.0.9",
+    "@angular-eslint/builder": "16.3.1",
+    "@angular-eslint/eslint-plugin": "16.3.1",
+    "@angular-eslint/eslint-plugin-template": "16.3.1",
+    "@angular-eslint/schematics": "16.3.1",
+    "@angular-eslint/template-parser": "16.3.1",
```

Comment:
Shouldn't these match angular's major version?

Reply:
Yes you're right, fixed it.